test/PrizePool.t.sol

```diff
@@ -158,22 +158,57 @@ contract PrizePoolTest is Test {
         assertEq(claimPrize(msg.sender, 0), 4.090909090909090318e18, "prize money for draw 2");
     }
 
-    function testGetVaultPortionWhenEmpty() public {
-        assertEq(SD59x18.unwrap(prizePool.getVaultPortion(address(this), 0, 1)), 0);
+    function testGetVaultPortion_WhenEmpty() public {
+        assertEq(SD59x18.unwrap(prizePool.getVaultPortion(address(this), 0, 0)), 0);
     }
 
-    function testGetVaultPortionWhenOne() public {
-        contribute(100e18);
-        assertEq(SD59x18.unwrap(prizePool.getVaultPortion(address(this), 1, 2)), 1e18);
+    function testGetVaultPortion_WhenOne() public {
+        contribute(100e18); // available draw 1
+        assertEq(SD59x18.unwrap(prizePool.getVaultPortion(address(this), 1, 1)), 1e18);
+    }
+
+    function testGetVaultPortion_WhenTwo() public {
```

Comment:
Why remove?

getVaultPortion when there is one observation and two observations is still important...esp testing the inclusive nature of (1,1) or (2,2) etc

Reply:
I renamed the `WhenEmpty`, `WhenOne`, and `WhenTwo` functions to add underscores to conform with the testing standard, and then updated the arguments with values that made sense for the new interface changes. These are still testing cases when there are one and two contributions, and the ranges are still inclusive of when the contributions should be available.

No tests were removed on this file.